src/Symfony/Bundle/FrameworkBundle/Templating/TemplateNameParser.php

```diff
@@ -72,4 +72,9 @@ public function parse($name)
 
         return $this->cache[$name] = $template;
     }
+
+    private function isAbsolute($path)
```

Comment:
Can you rename to `isAbsolutePath()` as this is the name we are using elsewhere.

Also, I'd prefer to use a more restrictive approach like done elsewhere:

```php
    private function isAbsolutePath($file)
    {
        return $file[0] === '/' || $file[0] === '\\'
            || (strlen($file) > 3 && ctype_alpha($file[0])
                && $file[1] === ':'
                && ($file[2] === '\\' || $file[2] === '/')
            );
    }

```